commands/user_test.go

```diff
@@ -53,6 +55,76 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
 	})
 }
 
+func (s *MmctlUnitTestSuite) TestSendPasswordResetEmailCmd() {
+	s.Run("Send one reset email", func() {
+		printer.Clean()
+		emailArg := "example@example.com"
+
+		s.client.
+			EXPECT().
+			SendPasswordResetEmail(emailArg).
+			Return(false, &model.Response{Error: nil}).
+			Times(1)
+
+		err := sendPasswordResetEmailCmdF(s.client, &cobra.Command{}, []string{emailArg})
+		s.Require().Nil(err)
```

Comment:
there is the [NoError](https://godoc.org/github.com/stretchr/testify/require#NoError) for this kind of thing, which communicates more clearly the intent